packages/connect/src/types/api/ethereum/index.ts

```diff
@@ -100,7 +100,7 @@ export const EthereumSignTypedDataMessage = Type.Object({
     domain: Type.Object({
         name: Type.Optional(Type.String()),
         version: Type.Optional(Type.String()),
-        chainId: Type.Optional(Type.Union([Type.Number(), Type.BigInt()])),
+        chainId: Type.Optional(Type.Union([Type.Number(), Type.BigInt(), Type.String()])),
```

Comment:
please add some standardized comments why this loosening is needed. it should be removed in next major version